docs/Ways-of-Working/Toolkit/Hit-Squads.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Hit Squads
+tags: 
+ - Default
+ - Hit-Squads
+---
+
+
+## Hit Squads
+
+A hit squad is a cross-team group of individuals focussed on solving *high priority*, *well-defined* *ways of working problems*.
```

Comment:
What are the benefits to restricting the scope of Hit Squads to "ways of working problems"? Aren't Hit Squads be a good option for tackling all (e.g. engineering, UX etc) high-priority, well-defined, cross-department problems?

Reply:
By ways of working problems, I mean everything relating to how we work, how we deliver value. The value delivery itself is performed by the delivery teams / support hub. This is a pretty broad categorisation, and looking at all of the hit squads that have previously existed / currently exist / had been proposed (that I am aware of), they almost all fit into this camp: Requirements management, BDD, Security awareness, Documentation, AI, RAID, DevEx, Design system automation, observability, success metrics & measurement.

The two exceptions that I am aware of that do not are: Disaster Recovery, and Accessibility Audit. The former was quickly assimilated into the support hub and I believe better suits the support hubs categories of work as they have now been defined. The latter I can't judge as I have had limited involvement, bit there doesn't appear to be a problem statement not any one specific problem being addressed (?).

We could broaden it out which would make it more flexible but may risk making it less clear, or just accept that there are times were we may use a process designed for WoW problems for other problems.